src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.serviceimpl.DataConverterImpl;
+import core.basesyntax.service.serviceimpl.FileReaderImpl;
+import core.basesyntax.service.serviceimpl.FileWriterImpl;
+import core.basesyntax.service.serviceimpl.ReportGeneratorImpl;
+import core.basesyntax.service.serviceimpl.ShopServiceImpl;
+import core.basesyntax.stategy.OperationStrategy;
+import core.basesyntax.stategy.OperationStrategyImpl;
+import core.basesyntax.stategy.handler.BalanceOperation;
+import core.basesyntax.stategy.handler.OperationHandler;
+import core.basesyntax.stategy.handler.PurchaseOperation;
+import core.basesyntax.stategy.handler.ReturnOperation;
+import core.basesyntax.stategy.handler.SupplyOperation;
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
```

Comment:
The 'Main' class should be placed in a separate package to follow the project structure guidelines.